File: code-coach/free/easy/hovercraft.cs
using System;

namespace Sololearn {
    class Program {
        static void Main() {
            int sales = int.Parse(Console.ReadLine());
            int invest = 10 * 2000_000 + 1000_000;
            long monthly = (long)sales * 3000_000;

            if (monthly < invest)
                Console.WriteLine("Loss");
            else if (monthly == invest)
                Console.WriteLine("Broke Even");
            else
                Console.WriteLine("Profit");
        }
    }
}

File: code-coach/free/easy/hovercraft.swift
if let input = readLine(), let sales = Int(input) {
    let invest = 10 * 2000_000 + 1000_000
    let monthly = sales * 3000_000

    if monthly < invest {
        print("Loss")
    }
    else if monthly == invest {
        print("Broke Even")
    }
    else {
        print("Profit")
    }
}